exasol_script_languages_developer_sandbox/templates/release_code_build.jinja.yaml

```diff
@@ -68,6 +68,8 @@ Resources:
                 Effect: Allow
               - Action:
                 - s3:PutObject
+                - s3:GetObject
```

Comment:
I guess this for the rename, maybe add this as comment

Reply:
I kind of inverted the documentation: I added all necessary actions to the `AwsAccess` class. I think that's better and more clear.